### PR TITLE
added algorithm to reverse_string.py and minimized reverse_word.py

### PR DESCRIPTION
**Pull request by dharan1011**
I have added another pythonic techniques to reverse a string named pythonic2 and minimized the algorithm in reverse_word.py as it showed inconsistency :-1: 
```
// revese_word.py 
Old Output : apizz like I and kim keon am I
Updated Outpult : pizza like I and kim keon am I 
```

### CODE REVIEW

**Comment by Biswa369** [comment]
Hi

File: string/reverse_words.py [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
-        elif i == n-1:
-            if start:
-                reverse(arr, start, i)
-        else:
-            if not start:
-                start = i
-    return "".join(arr)
-
+def reverse_words(str):
+	res = " "
```
Comment (ankit167) [inline string/reverse_words.py]: We can further minimize this function-
res = ' '.join(str.split()[::-1])

**Comment by dharan1011** [comment]
@keon isnt my request upto the mark?? :disappointed:

File: string/reverse_string.py [hunk 2]
```diff
@@ -19,7 +19,15 @@ def iterative(s):
 print(iterative(s))
 
 def pythonic(s):
-    r = list(reversed(s))
-    return "".join(r)
+	r = list(reversed(s))
```
Comment (ankit167) [inline string/reverse_string.py]: Would recommend to skip this function, as it does not illustrate the algorithm behind reversing a string.

File: string/reverse_words.py (hunk 1 above)
Comment (ankit167) [inline string/reverse_words.py]: The algorithm uses an extra variable to store the reversed string. A better algorithm would be to reverse the string in place.

**Comment by ankit167** [comment]
@dharan1011 Please resolve conflicts and make a fresh request.

File: string/reverse_words.py (hunk 1 above)
Comment (dharan1011) [inline string/reverse_words.py]: :+1: Yes ur suggestion seems right

**Pull request closed without merging**